cpp/ql/lib/semmle/code/cpp/rangeanalysis/new/RangeAnalysis.qll

```diff
@@ -0,0 +1,95 @@
+private import semmle.code.cpp.rangeanalysis.new.internal.semantic.analysis.RangeAnalysis
```

Comment:
We should probably add some simple QLDoc to this file. If nothing else, just to make Code Scanning happy.

Reply:
Added.